modules/con_adding-openshift-routes-as-a-traffic-manager-for-argo-rollouts.adoc

```diff
@@ -0,0 +1,23 @@
+:_mod-docs-content-type: CONCEPT
+
+[id="adding-openshift-routes-as-a-traffic-manager-for-argo-rollouts_{context}"]
+= Adding OpenShift Routes as a traffic manager for Argo Rollouts
```

Comment:
I would change the title of the page to 'Using Argo Rollouts to route traffic via OpenShift Routes'

Reply:
Fixed.